core/config.py

```diff
@@ -178,4 +178,15 @@
 
     # Will be set to viur.core.version.__version__ in viur.core.__init__
     "viur.version": None,
+
+    # Admin Parts
+    "admin.name": "Viur Administartion--Testr",
+    "admin.logo": "",
+    "admin.login.background": "/vi/login-background-2.jpg",
+    # URL for the Logo over the VI Login screen
+    "admin.login.logo": "/vi/logo_test.svg",
+    "admin.color.primary": "#d00f1c",
+    "admin.color.secondary": "#333333",
```

Comment:
These shouldn't be test values. We should use general values or set them `None`.